### PR TITLE
Implementetion add

### PR DESCRIPTION
**Pull request by alexansaa**
In this milestone, the following features have been implemented:
- Quotes component implemented
- Quotes component can fetch and receive data successfully
- Quotes component added to the parent component and rendered successfully
- Loading, error, and successful fetching process messages implemented

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @alexansaa, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, that linters are not running**

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by alexansaa** [comment]
@franclobo I am currently looking that Linters are running and correctly fixed
image

File: src/components/Quotes.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <p>Quote:</p>
+      <p>{GetElement(data)}</p>
+    </div>
+  );
```
Comment (mckent05) [inline src/components/Quotes.js]: - Good job done so far, but I am getting an error when running your project, a reason for this is when the `Quote` component mounts and calls the `GetElement` function, the useEffect block has not run yet hence no data is set to your `data` state. I suggest that you do not need the `GetElement` function. You can render your data directly in your component after it has been set in the state. Please fix this 🙏 
Your project:
4wywyw4

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @alexansaa, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good fetching data from the given API 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quotes.js [hunk 2]
```diff
@@ -44,25 +31,33 @@ export default function RederQuote() {
...
+        Quote:
+      </p>
+      <p>
+        {data[0].quote}
+      </p>
+      <p>
+        Author:
+      </p>
+      <p>
+        {data[0].author}
```
Comment (Johnadibe) [inline src/components/Quotes.js]: > - [ ] Hello @alexansaa, I am actually getting an error when running your project, a reason for this is when the Quote component mounts and you render your data directly in the component, the useEffect block has not run yet hence no data is set to your data state😌. To fix this, Kindly add "?" before the ". See the code below:👇
> 
>  ```
>  <div>
>       <p>Quote:</p>
>       <p>{data[0]?.quote}</p>
>       <p>Author:</p>
>       <p>{data[0]?.author}</p>
>     </div>
> ```
> 
> This is the error I get when I run our project:
> 
> cd
>

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
### Hi @alexansaa🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter errors✔️
- Descriptive PR✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @alexansaa 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**